citi-onboarding/iJunior-onepage
# Bugfix/frontend

## What I did

-Front-end elements adjustment
  - Correction of the position of the points
  - Background adjustment


| Mockup | Real |
|--------|------|
| <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123304875-da54d200-d4f5-11eb-887c-cbf23e4ffb45.png"> | <img width="632" alt="Screen Shot 2021-03-16 at 14 03 32" src="https://user-images.githubusercontent.com/52111341/123304675-9a8dea80-d4f5-11eb-8eb0-0d90c119fcf8.png"> |


## How to test

1. Open the terminal and go to the folder of this project

1. In terminal:
   1. `git checkout develop`
   1. `git pull`
   1. `git checkout bugfix/frontend`
   1. `git pull origin bugfix/frontend`
   1. `yarn install`
   1. `code .`
   1. `yarn start`

1. In your browser go to http://localhost:3000

1. Check if the component looks just like the mockup in deferent screen sizes 

1. In VScode open `/client/src`

1. Read the changed files section on GitHub 

1. Test the things that you think are worth testing, even the ones that are not in this description 

&nbsp;
***
**Time spent:** 20min
